src/tests/ffi.cpp

```diff
@@ -5279,6 +5279,124 @@ TEST_F(rnp_tests, test_ffi_keys_import)
     rnp_ffi_destroy(ffi);
 }
 
+static bool
+fake_key_invalid_length(const char *valid_key_path,
+    const char *invalid_key_0_path, const char *invalid_key_1_path, const char *invalid_key_2_path)
+{
+    FILE *fp;
+    size_t read_len;
+    unsigned char pkt[2048];
+    unsigned char pkt_bit;
+
+    unsigned char fake_pkt[2048];
```

Comment:
As we are moving to C++, could you please use std::string/std::vector here instead of static arrays/char * ?